meson.build

```diff
@@ -9,6 +9,7 @@ project(
 
 i18n = import('i18n')
 
+cairo_dep = dependency('cairo', version: '>= 1.0')
```

Comment:
I'm a bit in two minds if it's worth having this. Generally GTK+ should pull in a much more recent version, so I guess it just adds unnecessary verbosity.